fg/apps/orders/templates/orders/sign-mta.html

```diff
@@ -26,9 +26,10 @@ <h2>Sign your Material Transfer Agreement</h2>
   <div class="row">
     <div class="col-md-12">
       <p class="alert alert-info alert-dismissible fade show" style="margin-bottom:200px">A Material Transfer Agreement is needed to proceed with this order. Please fill out the fields for the organization name and address, along with the recipient name and address, 
-         <a href="https://biobricks.org/open-mta-become-a-signatory/" target="_blank">save and sign</a>
-         the Material Transfer Agreement document, and then <a href="#upload">upload the document</a> using the form at the bottom
-         of this page.</p>
+         <a href="https://biobricks.org/open-mta-become-a-signatory/" target="_blank">sign</a>
+         the Material Transfer Agreement document, and then <a href="#upload">upload a signed implementation letter</a> using the form at the bottom
+         of this page. DO NOT UPLOAD A COPY OF THE MASTER: to send materials, we require an implementation letter for those particular materials, 
```

Comment:
Maybe consider putting the all caps in bold (with <strong></strong>) if you want it to stick out?